docs/topics/proc_interacting-with-the-circuit-breaker-booster.adoc

```diff
@@ -42,7 +42,9 @@ If the remote endpoint is unavailable, the `name` service responds with an HTTP
 The state can be:
 ** _open_ : the circuit breaker is preventing requests from reaching the failed service,
 ** _closed_: the circuit breaker is allowing requests to reach the service.
-
+ifdef::vert-x[]
+** _half-open_: the circuit breaker is allowing a request to reach the service. If the request succeeds, the state of the service is reset to closed. If the request fails, the timer is restarted.
+endif::[]
```

Comment:
I believe there should be an empty line below this statement to make sure that the sentence on line 48 is not appended to the _half-open_ bullet point in the Vert.x guide.

Reply:
Dang nab it. Good catch.

Reply:
All set now (fingers crossed)